ts/features/design-system/core/DSFooterActionsSticky.tsx

```diff
@@ -141,7 +141,7 @@ export const DSFooterActionsSticky = () => {
           }
         }}
         animatedStyles={{
-          mainBlock: actionBlockAnimatedStyle,
+          mainBlock: actionBlockAnimatedStyle as any,
```

Comment:
Is this edit still useful? Tried to delete it and no TS error pops up

Reply:
nice catch, I think I changed before upgrading reanimated due to a compile error, maybe it's not needed after the version bump of reanimated

Reply:
addressed in [a925841](https://github.com/pagopa/io-app/pull/5864/commits/a925841192585380fbbd8449b70883a92f2f8d51)